src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java

```diff
@@ -154,6 +155,16 @@
     private static final Logger logger
         = Logger.getLogger(AbstractTranscriptPublisher.class);
 
+    /**
+     * Ascept for successful upload of transcript
```

Comment:
"Upload" refers to the execution of the external script after the call finishes, correct?

Reply:
yes